flask_inertia/inertia.py

```diff
@@ -32,9 +32,11 @@
 
 import os
 from http import HTTPStatus
-from typing import Any, Optional
+from typing import Any, Optional, Union
 
-from flask import Flask, Response, current_app, request
+from flask import Blueprint, Flask, Response, current_app, request
+from flask.sansio.app import App
+from flask.sansio.blueprints import BlueprintSetupState
```

Comment:
Why do you get this type in `sansio`  instead of dedicated module ?

```suggestion
from flask.blueprints import BlueprintSetupState
```

Reply:
auto import. fixed: [f5bdcdd](https://github.com/j0ack/flask-inertia/pull/25/commits/f5bdcdd9a7e4313f6a8d279dbf17d739a77ea5da)